master/static/srv/enrich_task_logs.py

```diff
@@ -119,17 +118,22 @@ def run(self) -> None:
             # Timeout met.
             self.ship()
 
-    @backoff.on_exception(  # type: ignore
-        lambda: backoff.full_jitter(SHIPPER_FAILURE_BACKOFF_SECONDS),
```

Comment:
what you implemented was a constant backoff, but what you are deleting [is a lot more involved](https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/)

Reply:
well, not that much more involved.  Looks like the full jitter algorithm is just:

```python
sleep = random_between(0, min(cap, base * 2**attempt))
```

Reply:
so, i looked into the backoff source code trying to figure out how to reimplement full_jitter, and it turns out we weren't implementing it correctly anyway. as is, this code does constant (but random) backoff. i consulted Jerry/Bradley and they said constant is fine for this method.